File: bindings/dart/flutter/colibri_flutter/windows/colibri_flutter_plugin.cpp
#include <flutter/plugin_registrar_windows.h>

// Minimal plugin registration. Colibri is used via Dart FFI and bundled colibri.dll.
void ColibriFlutterPluginRegisterWithRegistrar(
    FlutterDesktopPluginRegistrarRef registrar) {
  (void)registrar;
}

File: bindings/dart/flutter/colibri_flutter/macos/Classes/ColibriFlutterPlugin.swift
import FlutterMacOS

/// Minimal macOS plugin registration. Native Colibri library is loaded via Dart FFI.
public class ColibriFlutterPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    // No method channel; Colibri is used via Dart FFI and bundled libcolibri.dylib.
  }
}
